test/models/friendship_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class FriendshipTest < ActiveSupport::TestCase
```

Comment:
This test is incomplete as it is. We should test for:

1. Valid friendships -> Please make your test descriptions understandable. As you are using minitest please refer to the examples in the Rails Tutorial book on how to write good tests

2. Invalid friendships.